### ISSUE TITLE
Collection Artifacts contain errant files

### ISSUE THREAD

**Issue by sivel** [CiscoDevNet/ansible-meraki#148]
It appears as though the collection generated from this repo contains a full python virtual environment, as well as many other files that shouldn't be included in the build.

Some other examples:

* `.vscode/settings.json`
* `tests/output/junit/meraki_site_to_site_vpn-tbo31dqq-1585418559.9924767.xml`
* `.DS_Store`

See the attached file for a full list.

[cisco.meraki.txt](https://github.com/CiscoDevNet/ansible-meraki/files/4938392/cisco.meraki.txt)

**Comment by kbreit** [comment]
@sivel How do I tell the collection to not have the files which shouldn't be there?

**Comment by kbreit** [comment]
@sivel Could you take a look at #149 and let me know if it, at least at a glance, covers everything it should. Here's what I'm seeing when I extract a new tarball.

```
(venv) temp [2.0/build_ignore●] % tar -xvzf cisco-meraki-1.3.2.tar.gz
x MANIFEST.json
x FILES.json
x plugins/
x plugins/doc_fragments/
x plugins/doc_fragments/__init__.py
x plugins/doc_fragments/meraki.py
x plugins/action/
x plugins/inventory/
x plugins/module_utils/
x plugins/module_utils/network/
x plugins/module_utils/network/meraki/
x plugins/module_utils/network/meraki/__init__.py
x plugins/module_utils/network/meraki/meraki.py
x plugins/filter/
x plugins/modules/
x plugins/modules/meraki_organization.py
x plugins/modules/meraki_config_template.py
x plugins/modules/meraki_device.py
x plugins/modules/meraki_network.py
x plugins/modules/meraki_content_filtering.py
x plugins/modules/meraki_mr_l3_firewall.py
x plugins/modules/meraki_switchport.py
x plugins/modules/meraki_mx_site_to_site_firewall.py
x plugins/modules/meraki_site_to_site_vpn.py
x plugins/modules/meraki_mx_l3_firewall.py
x plugins/modules/meraki_vlan.py
x plugins/modules/meraki_nat.py
x plugins/modules/__init__.py
x plugins/modules/meraki_static_route.py
x plugins/modules/meraki_switch_storm_control.py
x plugins/modules/meraki_switch_stack.py
x plugins/modules/meraki_firewalled_services.py
x plugins/modules/meraki_webhook.py
x plugins/modules/meraki_admin.py
x plugins/modules/meraki_intrusion_prevention.py
x plugins/modules/meraki_management_interface.py
x plugins/modules/meraki_ssid.py
x plugins/modules/meraki_ms_link_aggregation.py
x plugins/modules/meraki_switch_access_list.py
x plugins/modules/meraki_mx_l7_firewall.py
x plugins/modules/meraki_malware.py
x plugins/modules/meraki_mx_uplink.py
x plugins/modules/meraki_syslog.py
x plugins/modules/meraki_snmp.py
```

**Comment by sivel** [comment]
That looks good. Thanks for the help. As of now the ansible-2.10.0a3 package contained the full venv referenced before.

I honestly don't know what the expectation is about delivering the tests too, but I'm fine with it this way.

**Comment by kbreit** [comment]
@sivel Do I need this as part of master to be picked up by your work?

**Comment by sivel** [comment]
@kbreit I am not specifically doing anything. It was completely accidental that I ran across this.

Until this is used to build the collection artifact, the Ansible 2.10 release will contain your virtualenv. I am not part of the team that owns that process, but I did make them aware of this.

**Comment by kbreit** [comment]
I will release a new 1.3.2 version today to remedy this. Thank you.

**Comment by kbreit** [comment]
I released v1.3.2. yesterday and that should have the files removed. If there are files in the package which shouldn't be there, let me know.
